target/bin/fail2ban

```diff
@@ -73,6 +77,11 @@ else
       fi
       ;;
 
+    ( 'log' )
+      LOG=/var/log/mail/fail2ban.log
+      [[ -f "${LOG}" ]] && cat "${LOG}"
```

Comment:
The log file should exist, when fail2ban is running. Not sure if `[[ -f "${LOG}" ]]` is necessary. Keep or remove?

Reply:
If there's no error handling, you'd probably communicate such a failure better from the `cat` error anyway?

Reply:
Just `cat`ing should be enough, agreed. Because now, the error is silently dropped, as @polarathene said.